doc/src/reference.xml

```diff
@@ -561,13 +561,15 @@ path-constant DATA : data/a.txt ;
       <varlistentry>
         <term><literal>cflags</literal></term>
         <term><literal>cxxflags</literal></term>
+        <term><literal>compileflags</literal></term>
```

Comment:
I do not understand how 'cflags' can mean command line options for both C and C++ when the various compiler toolsets clearly state that 'cflags' specifies command line options for C mode and that 'compileflags' specifies command line options for both C and C++ mode. Can you explain this to me ?